common/common.go

```diff
@@ -436,27 +508,34 @@ func RunBuild(verbose bool, dir, imageName, dockerfile string, buildArgs []strin
 	}
 
 	go func(done chan<- error) {
-		args := []string{
-			"build",
-			"-t", imageName,
-			"-f", dockerfile,
-		}
-		if noCache {
-			args = append(args, "--no-cache")
-		}
+		var dockerBuildCmdArgs []string
+		// Depending whether architecture list is passed or not trigger docker buildx or docker build accordingly
```

Comment:
We should test podman buildx as well.

Reply:
Currently not tested with podman, only tested with docker.